src/Symfony/Component/Debug/ErrorHandler.php

```diff
@@ -71,9 +71,9 @@ class ErrorHandler
     private $loggers = array(
         E_DEPRECATED => array(null, LogLevel::INFO),
         E_USER_DEPRECATED => array(null, LogLevel::INFO),
-        E_NOTICE => array(null, LogLevel::NOTICE),
-        E_USER_NOTICE => array(null, LogLevel::NOTICE),
-        E_STRICT => array(null, LogLevel::NOTICE),
+        E_NOTICE => array(null, LogLevel::WARNING),
+        E_USER_NOTICE => array(null, LogLevel::WARNING),
```

Comment:
You are right.
But I think it's better to log an error for a PSR logger. So user notices might be triggered by "old" third party library. That's why I think it's better so convert user notice to psr warning.

But I you really want psr notice, I can change it. Just tell me ;)